kite-data/kite-data-crunch/pom.xml

```diff
@@ -31,8 +31,20 @@
   </description>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/test/avro</directory>
```

Comment:
Why add src/test/avro to resources? Won't this include the schemas in the main jar?